nilearn/_utils/cache_mixin.py

```diff
@@ -30,6 +30,46 @@
 __CACHE_CHECKED = dict()
 
 
+def _check_memory(memory, verbose=False):
```

Comment:
A tiny docstring would really help. In particular detailing the accepted types of the memory argument, as this is non trivial.